src/Model/Quote/Checkout.php

```diff
@@ -75,19 +76,23 @@ public function __construct(
     public function getDeliveryOptions(array $forAddress = []): array
     {
         $this->hideDeliveryOptionsForProduct();
-        $packageType = $this->getPackageType();
-
-        if (isset($forAddress['countryId'])) {
-            $this->country = $forAddress['countryId'];
 
+        if (isset($forAddress['countryId'])
+            && !('US' === ($country = $forAddress['countryId']) && !isset($forAddress['city'])) // fake first request
```

Comment:
Use `AbstractConsignment::CC_US`?
```suggestion
            && !(AbstractConsignment::CC_US === ($country = $forAddress['countryId']) && !isset($forAddress['city'])) // fake first request
```